e2e/specs/create.project.spec.js

```diff
@@ -1,47 +1,30 @@
 const SingIn = require('../pages/SignIn');
-let username = 'hapsneeze';
-let password = 'test12345';
-describe('pivotal tracker page new project', () => {
+const APIrequest = require('../rest-api/RequestManager');
+const config = require('../../config.json');
+describe('pivotal tracker page create new project', () => {
 
     let dashboard;
-    let project;
     let projectData1 = {
-        name: 'test1',
-        account: 'TEST'
+        name: 'test wdio 1 -' + new Date().getMilliseconds(),
+        account: 'test'
     };
-    let projectData2 = {
-        name: 'test2',
-        account: 'testAccount',
-        privacy: 'public'
-    };
-    let projectData3 = {
-        name: 'test3',
-        privacy: 'private'
-    };
-    let projectData4 = {
-        name: 'test4',
-        account: 'newAccount',
-        privacy: 'public'
-    };
-
     before(() => {
-        dashboard = SingIn.loginAs(username, password);
+        dashboard = SingIn.loginAs(config.username, config.password);
     });
 
     it('should create a new private project with first account', () => {
-        project = dashboard.createProject(projectData1);
+        dashboard.clickCreateProjectButton();
+        let project = dashboard.createProject(projectData1);
         expect(`${projectData1.name} - Pivotal Tracker`).to.equal(project.getProjectName());
     });
-    it('should create a new public project with second account', () => {
-        project = dashboard.createProject(projectData2);
-        expect(`${projectData2.name} - Pivotal Tracker`).to.equal(project.getProjectName());
-    });
-    it('should not create a new private project with any account', () => {
-        project = dashboard.createProject(projectData3);
-        expect(`${projectData3.name} - Pivotal Tracker`).to.not.equal(project.getProjectName());
-    });
-    it('should create a new private project with a new account', () => {
-        project = dashboard.createProject(projectData4);
-        expect(`${projectData4.name} - Pivotal Tracker`).to.equal(project.getProjectName());
+
+    after(() =>{
+        let response = browser.call(() => {return APIrequest.GetRequest('projects');});
+        let projectID = Object.values(response.data).map((project) => {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this function to use "return" consistently. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS3801)